lib/shopify_api/context.rb

```diff
@@ -51,6 +52,7 @@ def setup(
         is_embedded:,
         session_storage:,
         logger: Logger.new($stdout),
+        host: ENV["HOST"] || "https://#{host_name}",
```

Comment:
Hmm, the naming doesn't feel right here, I wouldn't expect `host` to include a protocol.

Reply:
(maybe call it `host_with_scheme`?)

Reply:
I agree, the naming here isn't as precise as I'd hope. We use `host` name to imply the needed scheme. For that consistency I think we should leave it as is :(